transactions/btc.ts

```
@@ -901,6 +911,7 @@ export async function signOrdinalTransaction(ordinalTxParams: {
   const address = isRecover ? btcAddress : ordinalsAddress;
   const addressUtxos = await btcClient.getUnspentUtxos(address);
   const ordUtxo = getOrdinalUtxo(addressUtxos, ordinal);
+  const addressOrdinalsUtxos = await getOrdinalsUtxos(btcAddress);
   if (!ordUtxo) {
     // TODO: Throw error and not just the code
     // eslint-disable-next-line @typescript-eslint/no-throw-literal
```

Please throw the error here instead of the string and remove the 2 comments